mmpose/datasets/datasets/fashion/deepfashion_lower_dataset.py

```diff
@@ -0,0 +1,155 @@
+import os
+from collections import OrderedDict
+
+import numpy as np
+
+from mmpose.datasets.builder import DATASETS
+from .fashion_base_dataset import FashionBaseDataset
+
+
+@DATASETS.register_module()
+class DeepFashionLowerDataset(FashionBaseDataset):
+    """DeepFashion dataset (lower-body clothes) for fashion landmark detection.
+
+    `DeepFashion: Powering Robust Clothes Recognition
+    and Retrieval with Rich Annotations' CVPR'2016 and
+    `Fashion Landmark Detection in the Wild' ECCV'2016
+
+    The dataset loads raw features and apply specified transforms
+    to return a dict containing the image tensors and other information.
+
+    Fashion landmark indexes for lower-body clothes::
+
+        0: 'left waistline',
+        1: 'right waistline',
+        2: 'left hem',
```

Comment:
does hem belongs to both upper and lower?

Reply:
Yes, but not sure why.

Reply:
two types of hem maybe